subsys/nrf_security/src/drivers/cracen/cracenpsa/src/key_management.c

```diff
@@ -1302,37 +1302,22 @@ psa_status_t cracen_export_key(const psa_key_attributes_t *attributes, const uin
 		PSA_KEY_LIFETIME_GET_LOCATION(psa_get_key_lifetime(attributes));
 
 	if (location == PSA_KEY_LOCATION_CRACEN_KMU) {
-		/* The keys will already be in the key buffer as they got loaded their by a previous
-		 * call to cracen_get_builtin_key or cached in the memory.
-		 */
```

Comment:
As far as I can tell this comment is just wrong, there was no guarantee that this is true .